src/pages/layout/index.tsx

```diff
@@ -52,6 +54,7 @@ function walletsForNetwork(network: string) {
     okxWallet,
     martianWallet,
```

Comment:
probably push martian down due to lack of activity
fewcha is still actively developed so put bitget below?